router.php

```diff
@@ -56,6 +56,13 @@
     // Do nothing - assume the function source is being autoloaded.
 }
 
+if (class_exists(Google\Cloud\Storage\StreamWrapper::class)) {
+    // Register the "gs://" stream wrapper for Cloud Storage if the package
+    // "google/cloud-storage" is installed.
+    $storage = new Google\Cloud\Storage\StorageClient;
+    Google\Cloud\Storage\StreamWrapper::register($storage);
```

Comment:
Can developers re-register the `gs://` protocol if they wanted to customize the wrapper?

Reply:
Yes, they just need to do this:

```php
// unregister the automatically registered one 
stream_wrapper_unregister('gs');
```